.github/workflows/build-images-base.yaml

```diff
@@ -32,7 +45,7 @@ jobs:
         id: build-image
         uses: redhat-actions/buildah-build@v2
         with:
-          image: limitador-operator
+          image: ${{ env.OPERATOR_NAME }}
```

Comment:
nit: this is not necessary. I hardly can see OPERATOR_NAME value other than `limitador-operator`.

Reply:
It's just to make this workflow exportable, maybe after this PR move the reusable workflow somewhere that can be used by Authorino as well.. would be ideal to have just the one for Kuadrant operator too